LK Saarpfalz-Kreis Renamed

There's an error causing the tests to fail:

```
E         Extra items in the left set:
E         'LK Saarpfalz-Kreis'
E         Extra items in the right set:
E         'LK Saar-Pfalz-Kreis'
```

Looks like this Landkreis was renamed which leads to a check failing.

_Originally posted by @RobertRosca in https://github.com/oscovida/oscovida/issues/209#issuecomment-769668803_

--- a/oscovida/oscovida.py
+++ b/oscovida/oscovida.py
@@ -452,6 +452,11 @@ def germany_get_population() -> pd.DataFrame:
         .set_index('county')
     )
     population = population.rename(columns={"EWZ": "population"})
+    # see https://github.com/oscovida/oscovida/issues/210
+    # try to remove this if-clause and see if tests fail:
+    if "LK Saar-Pfalz-Kreis" in population.index:
+        population.loc['LK Saarpfalz-Kreis'] = population.loc['LK Saar-Pfalz-Kreis']
+        population = population.drop('LK Saar-Pfalz-Kreis')
     return population # type: ignore
 
 

--- a/tests/test_corona.py
+++ b/tests/test_corona.py
@@ -1,7 +1,6 @@
 import datetime
 import numpy as np
 import pandas as pd
-from pandas import DatetimeIndex
 import matplotlib.pyplot as plt
 import pytest
 import oscovida as c
@@ -321,6 +320,10 @@ def test_germany_get_population():
 
     pinneberg = germany.loc['LK Pinneberg'].population
     assert pinneberg > 30000
+
+    # https://github.com/oscovida/oscovida/issues/210
+    saarpfalz = germany.loc['LK Saarpfalz-Kreis'].population
+    assert saarpfalz > 130000
 
     aachen = germany.loc['StadtRegion Aachen'].population
     assert aachen > 500000

